UI/message-panel.tsx

```diff
@@ -562,7 +570,31 @@ function NoteCard(
         case NostrKind.DIRECT_MESSAGE:
             const profile = allUserInfo.get(event.pubkey)?.profile;
             return (
-                <div class={tw`px-4 my-1 py-2 border-2 border-[${PrimaryTextColor}4D] rounded-lg py-1 flex`}>
+                <div
+                    class={tw`px-4 my-1 py-2 border-2 border-[${PrimaryTextColor}4D] rounded-lg py-1 hover:bg-[${HoverButtonBackgroudColor}] cursor-pointer flex`}
+                    onClick={() => {
+                        let root: NostrEvent;
+                        root = event;
+                        if (event.parsedTags.root) {
+                            const res = eventSyncer.syncEvent(NoteID.FromHex(event.parsedTags.root[0]));
+                            if (res instanceof Promise) {
+                                return;
+                            }
```

Comment:
business logic like this should be handled in `update`

Reply:
ok